src/renderers/webgl/WebGLState.js

```diff
@@ -641,6 +640,7 @@ function WebGLState( gl, extensions, paramThreeToGL ) {
 
 		}
 
+		currentBlending = blending;
```

Comment:
Looks good. Just a single suggestion in context of code style: Please add a new line after this statement.